examples/image_magic_wand/tasks.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "id": 1202,
+    "data": {
+      "image": [
+        "https://storage.googleapis.com/brad-neuberg-label-studio/opensource/label-studio/examples/images/magic_wand_scale_1_20200902_015806_26_2235_1B_AnalyticMS_00750_00750.jpg",
```

Comment:
FYI these GCP buckets will have to be changed to `https://htx-misc.s3.amazonaws.com/opensource/label-studio/examples/images` and the assets uploaded there to reference from this example.